### DISCUSSION TITLE
Hey i have a doubt

### DISCUSSION THREAD

**Discussion by Deep-Svnit** [community/community#143375] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What should i practice leetcode vs codeforces

**Comment by THE-DEEPDAS** [comment, marked as answer]
It depends on your goals and current level of experience:

If you're aiming to improve competitive programming skills quickly and want to boost your Codeforces rating: Codeforces is a better choice. The platform’s frequent contests and challenging problems focus on problem-solving speed and accuracy, which can help you gain experience with high-pressure, time-constrained problem-solving. With a goal of reaching 1400, sticking with Codeforces could be beneficial, as it will directly improve your skills and rating on the platform where you want to progress.

If you’re interested in technical interview prep or want to develop a broader foundation in data structures and algorithms: LeetCode is great. It’s structured around common interview topics and problems, which are often more practical and less focused on fast-paced competition.

For well-rounded skills: You could combine both, dedicating time to Codeforces for contests and LeetCode for specific weaknesses or high-value topics in data structures and algorithms.
